api/src/main/java/org/openmrs/api/context/ServiceContext.java

```diff
@@ -207,19 +208,37 @@ public ObsService getObsService() {
 	}
 	
 	/**
-	 * @return condition related service
+	 * @return diagnosis related service
 	 * 
 	 * @since 2.2
 	 */
-	public ConditionService getConditionService() {
```

Comment:
Did you touch these methods by accident?

Reply:
This shifted from place when I added the getDiagnosisService

Reply:
Why?

Reply:
Let me fix it